docs/user/tutorials/01-120-aws-scheduled-nfs-backup.md

```diff
@@ -0,0 +1,57 @@
+# Create Scheduled Automatic RWX Volume Backups in Google Cloud
+
+This tutorial explains how to create scheduled automatic backups for ReadWriteMany (RWX) volumes in Google Cloud (AWS).
```

Comment:
Again, why "Google Cloud (AWS)"? AWS is not an acronym for Google Cloud. If this is supposed to mean something else, please try to rephrase it to avoid ambiguity. Isn't the title the same as of `./01-80-gcp-scheduled-nfs-backup.md`? Is this correct?

Reply:
It should be AWS, and I updated it now.